index.js

```diff
@@ -555,8 +555,6 @@ function checkoutTable (popup , tableCheckOutBtn ,  popupBillElementId ,  tableM
   let tableCheckOutBtnElement = document.getElementById(tableCheckOutBtn) ; 
       // Adding eventlistener to final checkout which will be called on btn click
```

Comment:
instead of duplicating the logic you write inside every table. write a function and reuse it.